src/components/landing/sponsors.astro

```diff
@@ -199,7 +199,7 @@ const sponsors = [
           <div class="mb-[64px] relative">
             <div class="flex items-center mb-6">
               <span class="block w-full h-[1px] bg-[#d3d2d6]" />
-              <span class="text-2xl shrink-0 font-semibold px-5">
+              <span class="text-2xl shrink-0 font-semibold px-5 max-w-full text-center">
```

Comment:
Could you update your commit title to: "sponsors: Repair broken overflow in tier names" instead?